autorest/validations.go

```diff
@@ -0,0 +1,359 @@
+package autorest
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+)
+
+// Constraint stroes constraint name, target field name
+// Rule and chain validations.
+type Constraint struct {
+
+	// Target Field name for validation.
+	Target string
```

Comment:
`field name` (also the comment on Constraint has upper/lower-case issues as well).